src/classes/Recipe.js

```diff
@@ -10,7 +10,7 @@ class Recipe {
 
   getIngredientNames(ingredientData) {
     return this.ingredients.map(ingredient => {
-      return ingredientData.find(data => ingredient.id === data.id).name.toLowerCase().split(' ');
+      return ingredientData.find(data => ingredient.id === data.id).name.toLowerCase().split(', ');
```

Comment:
Interesting, not sure I understand what this was solving yet, thanks for handling it, look forward to talking it out